src/app/pages/add-contact/addContact.component.html

```diff
@@ -74,12 +89,13 @@ <h1>Add Contact:</h1>
     </div>
 
     <div>
-      <button class="btn" (click)="submitForm()" [disabled]="isEmailInValid">
+      <button class="btn" (click)="submitForm()"
+        [disabled]="isEmailInValid || isMobileNumberInValid || isWorkNumberInValid || isHomeNumberInValid">
         Submit
       </button>
     </div>
 
-    <div class="err" [hidden]="!invalidAddContact">
+    <div class="err" [hidden]="invalidAddContact">
```

Comment:
Given that in your .ts file invalidAddContact, is always true, this will never show.